Source/WebCore/platform/graphics/cocoa/IOSurface.mm

```diff
@@ -651,30 +651,27 @@ static IntSize computeMaximumSurfaceSize()
     return DestinationColorSpace { colorSpaceCF };
 }
 
-IOSurface::Format IOSurface::formatForPixelFormat(PixelFormat format)
+IOSurface::Format IOSurface::formatForPixelFormat(ImageBufferPixelFormat format)
```

Comment:
This function is used by ImageBufferBackend only. Should not it be moved to ImageBufferPixelFormat.h:

```
constexpr PixelFormat convertToPixelFormat(ImageBufferPixelFormat format)
{
    ...
}

#if HAVE(IOSURFACE)
constexpr IOSurface::Format convertToIOSurfaceFormat(ImageBufferPixelFormat format)
{
    ...
}
#endif
```